.editorconfig

```diff
@@ -7,7 +7,7 @@ indent_style = space
 indent_size = 2
 trim_trailing_whitespace = true
 charset = utf-8
-max_line_length = 80
+max_line_length = 120
```

Comment:
@igor-ka I made this change since line length is set to `120` in the `tslint.json` file, but was 80 in editor config. (My editor shows a highlighted column on line 80 without this change, which is why I noticed it :P)